cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(5 * time.Second)
+
+	loginResponse := common.APIResponse{}
+	rawResponse := Login(C.CString("aaa"), C.CString(TestConfig.Account1.Password))
```

Comment:
I would recommend to use something more meaningful instead of `aaa`. Like `user1` or something.